src/test/java/com/google/cloud/tools/maven/it/RunMojoIntegrationTest.java

```diff
@@ -86,6 +97,34 @@ public void run() {
     assertEquals("Hello from the App Engine Standard project.", urlContent.toString());
     verifier.verifyErrorFreeLog();
     verifier.verifyTextInLog("Dev App Server is now running");
+    verifier.verifyTextInLog("Module instance " + expectedModuleName + " is running");
+  }
+
+  @SuppressWarnings("unused") // used for testRun()
```

Comment:
I don't understand why it would think it was unused, if it was in fact used? Is there some annotation somehwere that pulls it in?